### ISSUE TITLE
Unable to check name availability

### ISSUE THREAD

**Issue by somewhatnmc** [namecoin/electrum-nmc#192]
Consistent issue when checking name availability:

```
Traceback (most recent call last):
  File "electrum_nmc\electrum\gui\qt\main_window.py", line 3582, in check_name_availability
  File "electrum_nmc\electrum\gui\qt\main_window.py", line 2106, in 
  File "electrum_nmc\electrum\commands.py", line 146, in _run
  File "electrum_nmc\electrum\commands.py", line 118, in func_wrapper
  File "electrum_nmc\electrum\commands.py", line 1120, in name_show
  File "electrum_nmc\electrum\commands.py", line 1174, in name_show_single_try
Exception: The blockchain is still syncing

```

**Comment by binarypunk** [comment]
Exception: **The blockchain is still syncing**

**Comment by JeremyRand** [comment]
@somewhatnmc Okay, so obviously we need to be catching those exceptions.  That said, there's a potential secondary issue here, which is: is this Exception still happening for you when the blockchain is fully synced?  (You can look at the Network dialog to see the chain height of the servers and of your client; if your client is the same height as the servers, then you're fully synced; otherwise you are not, and this Exception is expected.)

**Comment by JeremyRand** [comment]
@somewhatnmc Also just to verify, did this show up with `nc3.3.8` tag, or master branch, or some other revision?  master branch is expected to take a while to sync, due to https://github.com/namecoin/electrum-nmc/issues/164

**Comment by somewhatnmc** [comment]
@JeremyRand this was using 3.3.8 binaries

**Comment by somewhatnmc** [comment]
Cannot reproduce issue currently, plausibly due to more Electrum-nmc servers being online. One, "Biolizard89" mentioned this may be the cause in "IRC".

**Comment by JeremyRand** [comment]
Okay, thanks @somewhatnmc .  I'll leave this issue open since the fact that the Exception is uncaught is clearly a bug that we need to fix, but it sounds like this bug isn't likely to surface much in typical use (except when there's trouble reaching a server).

**Comment by JeremyRand** [comment]
Possible explanation for the Exception being raised: https://github.com/namecoin/electrum-nmc/issues/220#issuecomment-610128670 .  If so then it should be fixed by 3.3.8.1.

Still leaving this issue open since catching the Exception is still important.
